spec/daru/io/exporters/sql_spec.rb

```diff
@@ -6,16 +6,11 @@
   let(:prepared_query) { double }
 
   it 'writes to an SQL table' do
-    expect(dbh)
-      .to receive(:prepare)
+    allow(dbh).to receive(:prepare)
```

Comment:
You got its suggestion wrong (the suggestion is questionable also, but let it be).
There are two ways of checking if something have received method:
1. `expect(something).to receive(:method)`
2. first stub with `allow(something).to receive(:method)` (or full spy/double), and then **check whether it was received** `expect(something).to have_received(:method)`

rubocop-rspec prefers second option. But in your current code, you check nothing -- you just allow it to receive... and then it maybe sent, or maybe not.

BTW, saharspec has [send_message](https://github.com/zverok/saharspec#send_messageobject-method-matcher) matcher, but I'm not insisting on its usage.